### PR TITLE
Image seeds

### LINKED ISSUE #4: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

__Story__:

Working on this application has been a lot of fun, right? You are fast
becoming a git expert, and an expert at working with a deployment
pipeline. Speaking of development and deployment pipelines, is it not pretty
annoying that everytime a new deployment is created it starts out with an empty
database? It seems like everytime a review application is created we need to
manually add a handful of images in order to manually test the
application. Would it not be great if each deployment was created with a
handful of existing images?

__Acceptance criteria__:
- [x] After running `rake db:setup` at least 20 unique images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 unique images on its homepage.

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by maiquynhtruong**
Closes #4

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,32 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
```
Comment (joshuasbates) [inline db/seeds.rb]: You can delete these default comments.  They are no longer needed.  :)

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,32 @@
...
+  'https://images.unsplash.com/photo-1558559294-e31a9d3bb51c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80',
+  'https://images.unsplash.com/photo-1558590319-7139feb0107a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80',
+  'https://images.unsplash.com/photo-1558590322-c45df0ab85ff?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80',
+  'https://images.unsplash.com/photo-1558478551-be297c7bb253?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+  'https://images.unsplash.com/photo-1558556515-2c1a28062c9d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80'
+]
+
+images_list.each do |url|
+  Image.create(url: url)
+end
```
Comment (joshuasbates) [inline db/seeds.rb]: This loop is fine, but you could also accomplish the same thing by passing an array of hashes tom `Image.create`, like:
```
image_seeds = [
{ url: http://some.url.com/some.jpg },
{ url: http://some.url.com/another.jpg },
...
]
Image.create(image_seeds)
```
This is just FYI, you don't need to update this.  :)

File: package.json [hunk 3]
```diff
@@ -1,12 +1,20 @@
 {
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
```
Comment (joshuasbates) [inline package.json]: You shouldn't have had to do this if you merged the image index story to master and then rebased this branch from master, right?  Aren't the same changes in this PR https://github.com/AppFolioOnboarding/image-sharer-maiquynhtruong/pull/15?

File: db/seeds.rb (hunk 1 above)
Comment (maiquynhtruong) [reply db/seeds.rb]: I deleted them :D

File: package.json (hunk 3 above)
Comment (maiquynhtruong) [reply package.json]: Yes. Yesterday I was waiting for your approval so I haven't rebased it. I rebased it now!

**Pull request merged**